tools/sdk-generation-pipeline/documents/task-engine/README.md

```diff
@@ -20,7 +20,7 @@ SDK Automation is launched in azure pipeline. It runs tasks in the following ste
 
 ### CodegenToSdkConfig
 This is type of file `./codegen_to_sdk_config.json` in sdk repo.
-The running environment of these scripts would be expected to be __Ubuntu 20.04__ on Azure Pipeline. This may change in the future. All the running script should be executable.
+The running environment of these scripts would be expected to be __Ubuntu 22.04__ on Azure Pipeline. This may change in the future. All the running script should be executable.
```

Comment:
Please revert this change because our docker image is based on ubuntu 20.04.

Reply:
@dw511214992 please check out [my comment here](https://github.com/Azure/azure-sdk-tools/pull/5004#discussion_r1055082694). Thanks!

Reply:
I created a follow-up issue. Please see this comment:
- https://github.com/Azure/azure-sdk-tools/pull/5004#issuecomment-1398536891